Maximum function nesting level of '256' reached!

Hello, 

I have just switched to the last release (1.3.2) after seeing that there was a fix for my problem (thank you!), and I am encountering a new type of error : 

When I load a list of fixtures, it stops working after a certain number. For example this loads fine (from id 2 to 3) :  

```
'App\Entity\InscriptionValidation':
    inscriptionValidation_1:
        inscription : '@inscription_1'
        validator : '@user_admin'
        status : 'OK'
    inscriptionValidation_{2..3}:
        inscription : '@inscription_<current()>'
        validator : '@user_*'
        status : 'En attente'

```
But if I want to go up to id 4, like so 

```
'App\Entity\InscriptionValidation':
    inscriptionValidation_1:
        inscription : '@inscription_1'
        validator : '@user_admin'
        status : 'OK'
    inscriptionValidation_{2..4}:
        inscription : '@inscription_<current()>'
        validator : '@user_*'
        status : 'En attente'

```

I get the stacktrace below.


```
[2021-01-05T09:27:11.670498+00:00] php.CRITICAL: Uncaught Error: Maximum function nesting level of '256' reached, aborting! {"exception":"[object] (Error(code: 0): Maximum function nesting level of '256' reached, aborting! at /srv/app/vendor/symfony/stopwatch/StopwatchEvent.php:236)"} []

In StopwatchEvent.php line 236:
                                                              
  [Error]                                                     
  Maximum function nesting level of '256' reached, aborting!  
                                                              

Exception trace:
  at /srv/app/vendor/symfony/stopwatch/StopwatchEvent.php:236
 round() at /srv/app/vendor/symfony/stopwatch/StopwatchEvent.php:236
 Symfony\Component\Stopwatch\StopwatchEvent->formatTime() at /srv/app/vendor/symfony/stopwatch/StopwatchEvent.php:226
 Symfony\Component\Stopwatch\StopwatchEvent->getNow() at /srv/app/vendor/symfony/stopwatch/StopwatchEvent.php:87
 Symfony\Component\Stopwatch\StopwatchEvent->start() at /srv/app/vendor/symfony/stopwatch/Section.php:119
 Symfony\Component\Stopwatch\Section->startEvent() at /srv/app/vendor/symfony/stopwatch/Stopwatch.php:106
 Symfony\Component\Stopwatch\Stopwatch->start() at /srv/app/vendor/symfony/doctrine-bridge/Logger/DbalLogger.php:43
 Symfony\Bridge\Doctrine\Logger\DbalLogger->startQuery() at /srv/app/vendor/doctrine/dbal/lib/Doctrine/DBAL/Logging/LoggerChain.php:39
 Doctrine\DBAL\Logging\LoggerChain->startQuery() at /srv/app/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php:1276
 Doctrine\DBAL\Connection->query() at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/Id/SequenceGenerator.php:80
 Doctrine\ORM\Id\SequenceGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
... ( a lot of the same lines)
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/IdGenerator.php:35
 Fidry\AliceDataFixtures\Bridge\Doctrine\IdGenerator->generate() at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:950
 Doctrine\ORM\UnitOfWork->persistNew() at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:1781
 Doctrine\ORM\UnitOfWork->doPersist() at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:1737
 Doctrine\ORM\UnitOfWork->persist() at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/EntityManager.php:610
 Doctrine\ORM\EntityManager->persist() at /srv/app/var/cache/dev/ContainerDThflVy/EntityManager_9a5be93.php:172
 ContainerDThflVy\EntityManager_9a5be93->persist() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/Persister/ObjectManagerPersister.php:76
 Fidry\AliceDataFixtures\Bridge\Doctrine\Persister\ObjectManagerPersister->persist() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Loader/PersisterLoader.php:92
 Fidry\AliceDataFixtures\Loader\PersisterLoader->load() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Loader/PurgerLoader.php:119
 Fidry\AliceDataFixtures\Loader\PurgerLoader->load() at /srv/app/vendor/theofidry/alice-data-fixtures/src/Loader/FileResolverLoader.php:73
 Fidry\AliceDataFixtures\Loader\FileResolverLoader->load() at /srv/app/vendor/hautelook/alice-bundle/src/Loader/DoctrineOrmLoader.php:159
 Hautelook\AliceBundle\Loader\DoctrineOrmLoader->loadFixtures() at /srv/app/vendor/hautelook/alice-bundle/src/Loader/DoctrineOrmLoader.php:125
 Hautelook\AliceBundle\Loader\DoctrineOrmLoader->load() at /srv/app/vendor/hautelook/alice-bundle/src/Console/Command/Doctrine/DoctrineOrmLoadDataFixturesCommand.php:163
 Hautelook\AliceBundle\Console\Command\Doctrine\DoctrineOrmLoadDataFixturesCommand->execute() at /srv/app/vendor/symfony/console/Command/Command.php:258
 Symfony\Component\Console\Command\Command->run() at /srv/app/vendor/symfony/console/Application.php:938
 Symfony\Component\Console\Application->doRunCommand() at /srv/app/vendor/symfony/framework-bundle/Console/Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at /srv/app/vendor/symfony/console/Application.php:266
 Symfony\Component\Console\Application->doRun() at /srv/app/vendor/symfony/framework-bundle/Console/Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /srv/app/vendor/symfony/console/Application.php:142
 Symfony\Component\Console\Application->run() at /srv/app/bin/console:43

hautelook:fixtures:load [-b|--bundle [BUNDLE]] [--no-bundles [NO-BUNDLES]] [-m|--manager MANAGER] [--append] [--shard SHARD] [--purge-with-truncate] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>

make: *** [Makefile:18: load-fixtures] Error 255

```




For information, here is my 0006_inscriptions.yaml, where I also had to scale down from 600 to 300 to make it load :  
```
'App\Entity\Inscription':
    abstract_inscription (template):
        user: '@user_*'
        event: '@event_*'
        program: '@program_1'
        status: '<randomElement([0, 1, 2, 3, 4])>'
        rank:  '<numberBetween(1, 10)>'
        origin: '30%? <sentence()>'
        creationDate: '<dateTime()>'
        lastChangeDate: '<dateTime()>'
    inscription_1 (extends abstract_inscription):
        user : '@user_user'
        event : '@event_demo_qualif_1_OneFormByMission'
        program : '@program_oneFormByMission'
        status : 0
        formBuilderForm : '21467'
        formBuilderSubmission : '61981'
        rank : 1
    inscription_{2..300} (extends abstract_inscription):
        program: '@program_company_2'
        formBuilderForm: '21467'
        formBuilderSubmission: '61981'
```

Does this look related to release 1.3.2 ? In the meantime, I'll keep my previously working version (1.1.2).

--- a/src/Bridge/Doctrine/Persister/ObjectManagerPersister.php
+++ b/src/Bridge/Doctrine/Persister/ObjectManagerPersister.php
@@ -62,7 +62,7 @@ class ObjectManagerPersister implements PersisterInterface
             // Check if the ID is explicitly set by the user. To avoid the ID to be overridden by the ID generator
             // registered, we disable it for that specific object.
             if ($metadata instanceof ORMClassMetadataInfo) {
-                if ($metadata->usesIdGenerator() && false === empty($metadata->getIdentifierValues($object))) {
+                if ($metadata->usesIdGenerator() && 0 !== count($metadata->getIdentifierValues($object)) && !$metadata->idGenerator instanceof IdGenerator) {
                     $metadata = $this->configureIdGenerator($metadata);
                 }
             } else if ($metadata instanceof ODMClassMetadataInfo) {

